troposphere/static/js/components/projects/detail/resources/ProjectDetails.react.js

```diff
@@ -101,10 +101,18 @@ define(function (require) {
     },
 
     onMoveSelectedResources: function () {
-      modals.ProjectModals.moveResources(
-        this.state.selectedResources,
-        this.props.project
-      );
+        var match = false;
+        this.state.selectedResources.forEach(function(sel){
+            if (stores.VolumeStore.getAttachedResources().indexOf(sel.get('uuid')) !== -1) {
+                match = true;
+            }
+        });
```

Comment:
Consider pulling the call to `stores.VolumeStore.getAttachedResources()` up and storing it in a variable. 

`.forEach( ... )` is giving us a looping construct, and you're looping through the `VolumeStore` as well. But we don't expect the `VolumeStore` to be changing underneath us. 

So you could consider something like this: 
```javascript
var match = false, 
    attachedResources = stores.VolumeStore.getAttachedResources();

this.state.selectedResources.forEach(function(sel){
    if (attachedResources.indexOf(sel.get('uuid')) !== -1) {
        match = true;
    }
});
```